app/views/main.scala.html

```diff
@@ -12,42 +12,77 @@
         <title>@title</title>
         <link href='//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700' rel='stylesheet' type='text/css'>
         <link rel="stylesheet" href="@routes.Assets.at("lib/normalize.css/normalize.css")">
-        <link rel="stylesheet" href="@routes.Assets.at("lib/foundation/css/foundation.min.css")">
         <link rel="stylesheet" href="@routes.Assets.at("css/main.min.css")">
```

Comment:
Avoid bringing in the .min files - makes things harder to debug. Minification will occur when we build a final distribution via the pipeline.